python/pylibvw.cc

```diff
@@ -842,13 +718,29 @@ uint32_t ex_get_cbandits_class(example_ptr ec, uint32_t i) { return ec->l.cb.cos
 float ex_get_cbandits_probability(example_ptr ec, uint32_t i) { return ec->l.cb.costs[i].probability; }
 float ex_get_cbandits_partial_prediction(example_ptr ec, uint32_t i) { return ec->l.cb.costs[i].partial_prediction; }
 
+py::tuple examples_get_cb_label_from_adf(ex_list& examples)
+{
+  auto it = std::find_if(examples.begin(), examples.end(), [](example_ptr& ex) { return !(ex->l.cb.costs.empty()); });
```

Comment:
this might not be enough to determine whether the examples have a label, see [here](https://github.com/VowpalWabbit/vowpal_wabbit/blob/master/vowpalwabbit/cb.cc#L109)

Reply:
Isn't there a label on every example that has a cost?